src/js/components/FileInput/utils/formatBytes.js

```diff
@@ -0,0 +1,45 @@
+const SI_CONVERSION_FACTOR = 1000;
+const IEC_CONVERSION_FACTOR = 1024;
+
+const getBrowser = () => {
+  const currentOS = ['Win', 'Linux', 'Mac'].find(
+    (v) => window.navigator.userAgent.indexOf(v) >= 0,
+  );
+
+  return currentOS;
+};
+
+const defaultFormat = (size) => {
+  const units = ['B', 'KB', 'MB', 'GB', 'TB'];
+  const factor = SI_CONVERSION_FACTOR;
+  let index = 0;
+  let num = size;
+  while (num >= factor && index < units.length - 1) {
+    num /= factor;
+    index += 1;
+  }
+  return `${num.toFixed(1)} ${units[index]}`;
+};
+
+const windowsFormat = (size) => {
+  const num = Math.ceil(size / IEC_CONVERSION_FACTOR);
+
+  function formatNumber(number) {
+    return Intl.NumberFormat().format(number);
+  }
+
+  return `${formatNumber(num)} KB`;
```

Comment:
Looks good and thanks for making the change. It might be nice to simplify this to
```jsx
return `${Intl.NumberFormat().format(num)} KB`;
```
since the formatNumber function is trivial and not used elsewhere.

Reply:
done!